evalml/automl/engine/engine_base.py

```diff
@@ -170,7 +172,7 @@ def train_and_score_pipeline(
     for i, (train, valid) in enumerate(
         automl_config.data_splitter.split(full_X_train, full_y_train)
     ):
-        if pipeline.model_family == ModelFamily.ENSEMBLE and i > 0:
+        if isinstance(pipeline.estimator, SklearnStackedEnsembleBase) and i > 0:
             # Stacked ensembles do CV internally, so we do not run CV here for performance reasons.
             logger.debug(
                 f"Skipping fold {i} because CV for stacked ensembles is not supported."
```

Comment:
Could you update the comment/debug message here since this is now only applicable for sklearn ensembles?